framework/mutex/FileMutex.php

```diff
@@ -137,6 +134,6 @@ protected function releaseLock($name)
      */
     protected function getLockFilePath($name)
     {
-        return $this->mutexPath . '/' . md5($name) . '.lock';
+        return $this->mutexPath . DIRECTORY_SEPARATOR . md5($name) . '.lock';
```

Comment:
Im considering '/' to be a best practice and to use DIRECTORY_SEPARATOR only when there is some matching to do.